### ISSUE TITLE
Migration with Mysqldump miss 'almost' of data

### ISSUE THREAD

**Issue by anhnongdan** [anhnongdan/bimax_ha_migration#4]
+ Migrate archive numeric and blob table from Feb -> June
+ Only June data show up. Other months' data is in DB but doesn't show on DB.

**Comment by anhnongdan** [comment]
Compare archive 2017_05 of pw1 (source) and pw2(dest) blob matched but numeric doesn't:

```
MariaDB [pw2]> select count(*) from piwik_archive_blob_2017_05;
+----------+
| count(*) |
+----------+
|    21219 |
+----------+
1 row in set (0.00 sec)

MariaDB [pw2]> select count(*) from piwik_archive_numeric_2017_05
    -> ;
+----------+
| count(*) |
+----------+
|      399 |
+----------+
1 row in set (0.00 sec)
```
```
MariaDB [pw1]> select count(*) from piwik_archive_numeric_2017_05
    -> ;
+----------+
| count(*) |
+----------+
|    12194 |
+----------+
1 row in set (0.01 sec)

MariaDB [pw1]> select count(*) from piwik_archive_blob_2017_05;
+----------+
| count(*) |
+----------+
|    21219 |
+----------+
1 row in set (0.01 sec)
```

**Comment by anhnongdan** [comment]
This is not what I want, something went wrong:

```
MariaDB [pw2]> select distinct(ts_archived) from piwik_archive_numeric_2017_04 where name like "%done%";
+---------------------+
| ts_archived         |
+---------------------+
| 2017-07-03 16:53:11 |
| 2017-07-03 16:53:12 |
| 2017-07-03 16:53:13 |
| 2017-07-03 16:53:14 |
| 2017-07-03 16:53:15 |
| 2017-07-03 16:53:16 |
| 2017-07-03 16:53:17 |
| 2017-07-03 16:53:18 |
| 2017-07-03 16:53:19 |
| 2017-07-03 16:53:20 |
| 2017-07-03 16:53:21 |
| 2017-07-03 16:53:22 |
| 2017-07-03 16:53:23 |
| 2017-07-03 16:53:24 |
| 2017-07-03 16:53:25 |
| 2017-07-03 16:53:26 |
| 2017-07-03 16:53:27 |
| 2017-07-03 16:53:28 |
| 2017-07-03 16:53:29 |
| 2017-07-03 16:53:30 |
| 2017-07-03 16:53:31 |
| 2017-07-03 16:53:32 |
| 2017-07-03 16:53:33 |
| 2017-07-03 16:53:34 |
| 2017-07-03 16:53:35 |
```

```
MariaDB [pw1]> select distinct(ts_archived) from piwik_archive_numeric_2017_04 where name like "%done%";
+---------------------+
| ts_archived         |
+---------------------+
| 2017-04-01 17:00:09 |
| 2017-04-01 17:00:15 |
| 2017-04-02 17:00:12 |
| 2017-04-02 17:00:20 |
| 2017-04-03 16:50:05 |
| 2017-04-03 17:00:10 |
| 2017-04-03 17:00:11 |
| 2017-04-03 17:00:15 |
```

**Comment by anhnongdan** [comment]
--result-file of mysqldump repeat through blob and numeric data, the last dump actually override the first.

Fixed: 3bf7605834c494c523192f53e8f768b45b4ed00b
